.drone.yml

```diff
@@ -143,6 +143,14 @@ pipeline:
     when:
       branch: [dev]
       event: [push]
+    
+  build_docs:
```

Comment:
Is it worth putting this in the `build-artifacts` group to save time?

Reply:
Yes, done